src/app/registration/facility-select/facility-select.component.html

```diff
@@ -70,7 +70,7 @@
               Pass availability -
               <span [ngClass]="timeConfig.AM.text">{{ timeConfig.AM.text }}</span>
             </p>
-            <p class="card-text">7am&ndash;12pm (Depart by noon)</p>
+            <p class="card-text">{{to12hTimeString(defaultAMOpeningHour)}}&ndash;12pm (Depart by noon)</p>
```

Comment:
I think this is still wrong if a custom facility opening hour has been set in admin, but it's an improvement on what was there 👍